charts/logging/loki/values.yaml

```diff
@@ -13,21 +13,8 @@
 # limitations under the License.
 
 loki:
-  image:
-    repository: docker.io/grafana/loki
   fullnameOverride: loki
 
-  affinity: {}
```

Comment:
All settings related to the Loki STS (affinities, tolerations, podAnnotations, ...) have been moved under `loki.singleBinary` upstream and so we do the same structural change here.